### ISSUE TITLE
Add option to be able to take captcha/cloudfare

### ISSUE THREAD

**Issue by PondelekB** [LNReader/lnreader#873]
### Describe your suggested feature

Add option to be able to take captcha/cloudfare test when searching novels on sources with this protection enabled 

### Other details

On some reader apps there is a button that allows you to pass the human authentication test to be able to read on sources with said protection 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with a source, I should be opening an issue in the [sources repository](https://github.com/LNReader/lnreader-sources/issues/new/choose).
- [X] I have updated the app to version **[1.1.18](https://github.com/LNReader/lnreader/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by rajarsheechatterjee** [comment]
You can open the source in WebView to bypass the Cloudflare protection.

**Comment by PondelekB** [comment]
> You can open the source in WebView to bypass the Cloudflare protection. 

Not entirely, coz i cant add an title to the library if i cant get past cloud flare and there is no option for opening webview there so nothing changes

**Comment by rajarsheechatterjee** [comment]
You can click the globe icon on top right to open WebView. 

![Screenshot_2024-01-05-23-13-21-73_3c1c179116822eec69c21805387f0489.jpg](https://github.com/LNReader/lnreader/assets/57822170/975b00f0-2b8f-4d91-b8e6-7569fc3cec95)

**Comment by PondelekB** [comment]
> You can click the globe icon on top right to open WebView. 
> 
> ![Screenshot_2024-01-05-23-13-21-73_3c1c179116822eec69c21805387f0489.jpg](https://github.com/LNReader/lnreader/assets/57822170/975b00f0-2b8f-4d91-b8e6-7569fc3cec95)
> 
> 

I know that but a few sources check the cloudflare every time the site is opened and don't do it only when the child sites are used so it doesn't allow this

**Comment by rajarsheechatterjee** [comment]
> I know that but a few sources check the cloudflare every time the site is opened and don't do it only when the child sites are used so it doesn't allow this

How does it not allow it? The searchbar and globe icon are still there if there's a Cloudflare check on this screen.

**Comment by PondelekB** [comment]
> The

Yes but after passing the cloudflare you have to close webview to add issue to the library and the cloudflare is reset

**Comment by samanta-widjaja** [comment]
I also have the same problem. before i update my library, i open the webview, but it still show message can't be update. it happens on novel updates

**Comment by shotwister** [comment]
Isn't there a way to change user string?

**Comment by B10Reaper** [comment]
Still seems to be no way to change the userstring or somehow get past the cloudflare loop when opening in webview.
Even when I could pass it in webview, it seemed to reset when I went to check it out in the app.
Anything to this? Is the app just abandoned now?

**Comment by Soopyboo32** [comment]
> Still seems to be no way to change the userstring or somehow get past the cloudflare loop when opening in webview. Even when I could pass it in webview, it seemed to reset when I went to check it out in the app. Anything to this? Is the app just abandoned now?

You can change user agent, its in settings > advanced

**Comment by B10Reaper** [comment]
Where?
(https://github.com/user-attachments/assets/760e22c4-8d58-4b39-a2aa-89f88c09e223)

**Comment by CD-Z** [comment]
You are on an old version. In that version this page wasn't scrollable so you can't see the option if your screen is too small.
